test/functional/device/motion-e2e-specs.js

```diff
@@ -1,61 +1,63 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import { MOCHA_TIMEOUT, initSession, deleteSession } from '../helpers/session';
-import { SETTINGS_CAPS } from '../desired';
+import { SETTINGS_CAPS, amendCapabilities } from '../desired';
 
 
 const PREDICATE_SEARCH = '-ios predicate string';
 
 chai.should();
 chai.use(chaiAsPromised);
 
-describe('ReduceMotion', function () {
+describe('ReduceMotion', function() {
   this.timeout(MOCHA_TIMEOUT);
 
-  let caps = Object.assign({usePrebuiltWDA: true}, SETTINGS_CAPS);
+  let caps = Object.assign({ usePrebuiltWDA: true }, SETTINGS_CAPS);
```

Comment:
consider using amendCapabilities instead. It is always better if capabilities are immutable